.travis.yml

```diff
@@ -8,13 +8,13 @@ env:
 
 before_install:
   - docker network create mynet
-  - docker run --net mynet --name my_solr -d -p 8983:8983 -t solr:7.1-alpine
-  
+  - docker run --net mynet --name my_solr -d -p 8983:8983 -t solr:7.1-alpine -DzkRun -Denable.runtime.lib=true
```

Comment:
here you need to add the mount option so that the BioSolr Jar is available to the server, probably mounting it in a directory that it is within the solr classpath.

Reply:
This should fit with what you have here: https://github.com/ebi-gene-expression-group/index-scxa/blob/feature/162418601-scxa-analytics-v3-with-biosolr/bin/create-scxa-analytics-biosolr-lib.sh#L9

and that directory should be written based on server paths, not the client, as it will be interpreted by the server.